package.json

```diff
@@ -80,7 +80,7 @@
     "jest": "^27.0.4",
     "lint-staged": "^10.5.4",
     "lodash": "^4.17.21",
-    "next": "^11.1.2",
```

Comment:
You would need to also increase the peer-dependency in the package.json to make it actually work for projects using next-page-tester.